StarVMC/xgeometry/.gitignore

```diff
@@ -0,0 +1 @@
+*.age
```

Comment:
Not quite (there is one file which should not be ignored) but I'll fix after the PR is accepted.